beakerx/beakerx/handlers.py

```diff
@@ -12,34 +12,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from notebook.utils import url_path_join
-from notebook.base.handlers import APIHandler, IPythonHandler
-
-import json
-from tornado import web
-from .environment import *
 import beakerx
-import tornado
+import json
 import os
+import tornado
+import zmq
+from notebook.base.handlers import APIHandler, IPythonHandler
+from notebook.utils import url_path_join
+from tornado import web
+
 from .beakerx_autotranslation_server import start_autotranslation_server
+from .environment import *
 
 
 class BeakerxRestHandler(APIHandler):
+
     def data_received(self, chunk):
         pass
 
     @web.authenticated
     @tornado.web.asynchronous
     def post(self):
 
-        def handle_response(response):
-            self.finish(response.body)
-
         data = tornado.escape.json_decode(self.request.body)
         content = json.dumps(data)
         params = json.loads(content)
+
+        type = params['type']
```

Comment:
This would raise a KeyError that breaks other post methods, like the spark interrupt button. Use dict.get instead.